src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -1380,11 +1380,11 @@ void MarkEntireAssembly (AssemblyDefinition assembly)
 			foreach (TypeDefinition type in module.Types)
 				MarkEntireType (type, new DependencyInfo (DependencyKind.TypeInAssembly, assembly));
 
-			foreach (ExportedType exportedType in module.ExportedTypes) {
-				MarkingHelpers.MarkExportedType (exportedType, module, new DependencyInfo (DependencyKind.ExportedType, assembly));
-				MarkingHelpers.MarkForwardedScope (new TypeReference (exportedType.Namespace, exportedType.Name, module, exportedType.Scope));
-			}
```

Comment:
I must admit I don't understand why we're removing marking of type forwarders... the rest of the changes in this method make sense, but this one I don't get.

Reply:
The forwarders are marked by the `TypeReferenceMarker` at https://github.com/dotnet/linker/pull/2276/files#diff-279c983ce836c70ed323727a69c23d650e18e177e3ec103cf15805ad69e9e017R151-R152.

I can move that back into this method to reduce the delta if you prefer.

Reply:
I would prefer we don't remove anything from the code which is there - ideally we should just add. I know that this should be the same... but still.

Reply:
Done!